### ISSUE TITLE
EM doesn't detect internet connectivity when using USB tethering.

### ISSUE THREAD

**Issue by TheShitcoiner** [vladimiry/ElectronMail#237]
I'm using Easytether to tether Android 9.0 to Kubuntu 19.10 laptop. ElectonMail works fine when I switch to wifi tethering but USB tethering is always faster than wifi. 

Easytether website: http://www.mobile-stream.com/
Guide: http://www.mobile-stream.com/easytether/android_faq.html#linuxsetup
Drivers: http://www.mobile-stream.com/easytether/drivers.html

**Comment by vladimiry** [comment]
Does the regular Chromium browser detect the internet connectivity when using USB tethering (ideally you try version close to Chromium 78.0.3904.130 as this version comes with app v4.2.0)?

**Comment by TheShitcoiner** [comment]
Yes. EM shows that empty dark screen until an internet connection has been established. 

![Screenshot_20200126_120511](https://user-images.githubusercontent.com/49177109/73139852-fbcb3900-4037-11ea-8dbd-0f4f736fb6d2.png)

**Comment by TheShitcoiner** [comment]
**Problem only exists at start**
I switched to wifi tethering to log in then back to USB tethering (disabled wifi). Once logged in, I can send and recieve emails.

**Comment by vladimiry** [comment]
The app doesn't allow interaction with online email account until internet connection got detected (but local database mode is available in offline mode if activated for the account). The app detects online state by pinging the `window.navigator.onLine` property (and in some cases listening for respective events). This property comes from Chromium which comes with Electron. So it looks like an upstream issue. But let's keeps the issue open, maybe one day the connection detection logic will be rethinked.
